Dalamud.Common/DalamudStartInfo.cs

```diff
@@ -17,38 +17,6 @@ public DalamudStartInfo()
         // ignored
     }
 
-    /// <summary>
-    /// Initializes a new instance of the <see cref="DalamudStartInfo"/> class.
-    /// </summary>
-    /// <param name="other">Object to copy values from.</param>
-    public DalamudStartInfo(DalamudStartInfo other)
```

Comment:
No longer used anywhere? (non-blocking, just asking)

Reply:
C# records have an implicit constructor doing the same; even without that, `new DalamudStartInfo(new DalamudStartInfo())` compiles.